Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM mcr.microsoft.com/dotnet/sdk:6.0 
 ENV test_file="core/Akka.Streams.Tests/Akka.Streams.Tests.csproj"
+ENV test_filter=""
```

Comment:
When there is only the need in some specific tests. The`run_count` makes this an interesting to do.